pkg/operation/common/types.go

```diff
@@ -131,6 +131,9 @@ const (
 	// is accepted.
 	GardenerDeletionProtected = "gardener.cloud/deletion-protected"
 
+	// ETCDSecretsEncryptionConfigDataName is the name of ShootState data entry holding the current key and encryption state used to encrypt shoot secrets
+	ETCDSecretsEncryptionConfigDataName = "etcdSecretsEncryptionConfiguration"
```

Comment:
```suggestion
	ETCDEncryptionConfigDataName = "etcdSecretsEncryptionConfiguration"
```

Remove "secrets" - maybe we will encrypt even more in the future